evaluation/check_predict.py

```diff
@@ -8,14 +9,28 @@
 from utils.evaluation import evaluate, print_stats
 import pickle
 
-# load model
+# script purpose and usage is explained 
+
+description = """   This script is made to see the predictions done by lodbrok model 
+                    against the 'spambrainz_dataset_predict.pickle' dataset, the label 
+                    mentioning whether the account is spam or not isn't consider while 
+                    loading the dataset. The predictions are classified such that,
+                    if prediction is closer to 1 it is a spam account and if it is closer 
+                    to 0 then it is non_spam account.
+                """
+
+parser = argparse.ArgumentParser(description=description)
+
+args = parser.parse_args()
+
```

Comment:
Please move it ahead of `sys.path.append("..")` or maybe even just after `import argparse` so that it is shown when executed from the wrong directory.